libraries/Microsoft.Bot.StreamingExtensions/Payloads/Assemblers/ContentStreamAssembler.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+
+namespace Microsoft.Bot.StreamingExtensions.Payloads
+{
+    internal class ContentStreamAssembler : PayloadAssembler
+    {
```

Comment:
Why would a Content Stream Assembly inherit from a Payload Assembler yet emit Payload Streams? 

Shouldn't this be a PayloadStreamAssembler?